cfme/containers/provider/__init__.py

```diff
@@ -228,18 +228,16 @@ def num_project(self):
     @num_project.variant('ui')
     def num_project_ui(self):
         view = navigate_to(self, "Details")
-        return int(view.entities.summary("Relationships").get_text_of("Projects"))
+        return int(view.entities.summary("Relationships").get_text_of("Container Projects"))
 
     @variable(alias='db')
     def num_service(self):
         return self._num_db_generic('container_services')
 
     @num_service.variant('ui')
     def num_service_ui(self):
-        if self.appliance.version < "5.7":
-            name = "Services"
-        else:
-            name = "Container Services"
+        name = VersionPick({Version.lowest(): 'Services',
```

Comment:
5.9 is our lowest supported version at the moment, checkout #7600.

You can collapse these to the 5.9  string only, or a version pick for 5.9+